sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/administration/MetricsAdvisorAdministrationAsyncClient.java

```diff
@@ -75,18 +75,64 @@
 import static com.azure.core.util.FluxUtil.withContext;
 
 /**
- * This class provides an asynchronous client that contains all the operations that apply to Azure Metrics Advisor.
- * <p><strong>Instantiating a asynchronous Metrics Advisor Administration Client</strong></p>
- * <!-- src_embed com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationAsyncClient.instantiation -->
+ * <p>This class provides asynchronous client to connect to the Metrics Advisor Azure Cognitive Service.</p>
+ * <p>This client provides asynchronous methods to perform:</p>
+ * <ol>
+ *     <li>Connect to a variety of data sources, Metrics Advisor can connect to, and ingest multi-dimensional metric data
+ *     from many data stores, including: SQL Server, Azure Blob Storage, and MongoDB. Use
+ *     {@link com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationClient#createDataFeed(DataFeed)}
+ *     method to add your respective data source.</li>
+ *     <li>Customize anomaly detection configuration to detect anomalies for your needs using the
+ *     {@link com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationClient#createDetectionConfig(String, AnomalyDetectionConfiguration)}
+ *     method.</li>
+ *     <li>Add real-time notification through multiple channels. Configure hooks for multiple alerting and detection
+ *     configuration using the
+ *     {@link com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationClient#createHook(NotificationHook)}
+ *     method./li>
+ * </ol>
+ *
+ * <p>Service clients are the point of interaction for developers to use Azure Metrics Advisor.
+ * {@link com.azure.ai.metricsadvisor.MetricsAdvisorClient} is the synchronous service client and
+ * {@link com.azure.ai.metricsadvisor.MetricsAdvisorAsyncClient} is the asynchronous service client.
+ * The examples shown in this document use a credential object named DefaultAzureCredential for authentication, which is
```

Comment:
Below we have a sample that also shows sample with `MetricsAdvisorKeyCredential`, we can consider mentioning it here.